mypy.ini

```diff
@@ -1,6 +1,11 @@
 [mypy]
 python_version = 3.9
```

Comment:
I believe we should use `python_version = 3.7` here, since it's the lowest supported version for the project. To make typing more powerful and annotations easier, you can add an additional dependency: https://pypi.org/project/typing-extensions/3.10.0.2/ (this exact version, to be bumped to https://pypi.org/project/typing-extensions/4.4.0/ in near future).

Reply:
It probably makes sense to change this python 3.9 to either 3.7 or 3.10. Not sure what @fingon  thinks about this. 

About the suggestion of checking python 3.11 discussed in the daily today, apparently the snappy package does not yet support 3.11 and from this issue it seems likely that they might never support it since the current maintainer is suggesting moving to cramjam... So moving Rohmu to python3.11 might have that issue.

If we need to change this I would prefer moving to 3.10 since 3.7 is EOL in ~1.5 months, and we should probably make sure that the unittest we at least import all modules (to catch runtime errors due to annotation definitions).
I will add the typing extensions module in the requirements explicitly.

Reply:
Okay, the typing_extensions package is already provided because it is required by a number of other packages:
```
$ pip show typing_extensions
Name: typing_extensions
Version: 4.5.0
Summary: Backported and Experimental Type Hints for Python 3.7+
Home-page: 
Author: 
Author-email: "Guido van Rossum, Jukka Lehtosalo, Łukasz Langa, Michael Lee" <levkivskyi@gmail.com>
License: 
Location: /home/giacomo.alzetta/Projects/rohmu/venv/lib/python3.10/site-packages
Requires: 
Required-by: azure-core, azure-storage-blob, google-api-python-client-stubs, mypy, mypy-boto3-sqs, pydantic
```
I believe it makes sense to include it explicitly. I would directly use version 4.5.0 which still supports python3.7.

Reply:
3.7 is really soon EOL (c.f. https://endoflife.date/python) so I wouldn't go for that at least, 3.8 might make sense though.

Reply:
IMHO, python3.7 is getting EOLed so I wouldn't want to move back there and next month having to change this again. For now I'm leaving the weird setup. In a next step we should probably move to python3.10.

For Python3.11 there is an issue with the snappy dependency which will need to be resolved...